sdcm/nemesis.py

```diff
@@ -2613,6 +2613,65 @@ def disrupt_mgmt_backup_specific_keyspaces(self):
     def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
+    def disrupt_mgmt_restore(self):
+        def get_minimum_free_space():
+            minimum_space = 99999999999999
+            for node in self.cluster.nodes:
+                result = node.remoter.run("df -al | grep '/var/lib/scylla'")  # Size in KB
```

Comment:
it may be a bit problematic, if we run `nemesis_during_prepare`, because could be a situation where we run this nemesis before we have the data written, so it is possible that we may over use the real free space for the cluster

also, how much is the max threshold for disk utilization in your calculation (we should never pass 80%, but in some cases, >50% can already be a problem